behaviour_and_protocol/values.md

```diff
@@ -27,15 +27,24 @@ They are set up as "X over Y" to make it clear that we value both, but aim to pr
 
 ## Trust
 - Team over individuals
+  - Individual difficulties are team difficulties - include the team in figuring them out
 - Autonomy over control
+    - Build a process that caters to self-reliance, not one that calls for control
 - Transparency over appearances
+  - Share everything, also the things you are not proud of yet
```

Comment:
This is one aspect of it. Transparency within the team.
Another, that would be nice to cover, is transparency toward the clients.

Reply:
I read this as transparency with both clients and colleagues. 
If we follow the "on site customer"-rule in XP, our clients should be considered as part of the team